src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedNewTests.cs

```diff
@@ -0,0 +1,884 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    [CompilerTrait(CompilerFeature.TargetTypedNew)]
+    public class TargetTypedNewTests : CSharpTestBase
+    {
```

Comment:
>{ [](start = 4, length = 1)

I have not looked at the tests deeply yet. I'll finish tomorrow.
Things look great so far. Thanks